### PR TITLE
Reorg tiering policy sections into manage tiering

### LINKED ISSUE #3508: [Docs RFC] Reorg tiering policy sections into Manage tiering
Ticket for following up on this one:

> How about we combine https://docs-dev.timescale.com/docs-upd2/use-timescale/upd2/data-tiering/enabling-data-tiering/, https://docs-dev.timescale.com/docs-upd2/use-timescale/upd2/data-tiering/creating-data-tiering-policy/ and this doc into one page called Manage data tiering or some such like?

`
https://github.com/timescale/docs/pull/3496#issuecomment-2405058858

### PR DESCRIPTION
**Pull request by atovpeko**

### CODE REVIEW

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 1]
```diff
@@ -9,27 +9,24 @@ cloud_ui:
         - [services, :serviceId, overview]
 ---
 
-# Tier data to the object storage tier
+# Manage tiering
 
-Enable tiered storage to begin migrating rarely used data from Timescale's standard high-performance storage tier
-to the object storage tier to save on storage costs. 
+You use tiered storage to save on storage costs. Specifically, you can migrate rarely used data from Timescale's standard high-performance storage to the object storage. With tiered storage enabled, you then either create automated tiering policies or manually tier and untier data.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]:
```suggestion
You use tiered storage to save on storage costs. Specifically, you can migrate rarely used data from Timescale's standard high-performance storage to the object storage. After you enable tiered storage, you then either create automated tiering policies or manually tier and untier data.
```

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 2]
```diff
@@ -9,27 +9,24 @@ cloud_ui:
...
+## Enable tiered storage
 
-<Procedure>
+You enable tiered storage from the `Overview` tab in Console.
 
-### Enabling tiered storage
+<Procedure>
 
-1.  In the Timescale console, from the `Services` list, click the name of
+1.  In Timescale Console, from the `Services` list, click the name of
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]:
```suggestion
1.  In [Timescale Console][services-portal], select the service to modify. 
      You see the the Overview section.  
       
1. Scroll down, then click `Enable tiered storage`.
   ![Enable tiered storage](https://assets.timescale.com/docs/images/console-enable-tiered-storage.png)
    When tiered storage is activated, you see the amount of data in the tiered object storage.  
   

```

[services-portal]: https://console.cloud.timescale.com/dashboard/services

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 3]
```diff
@@ -9,27 +9,24 @@ cloud_ui:
...
     the service you want to modify.
-1.  In the `Overview` tab, locate the `Tiered Storage` card, and click
+1.  In the `Overview` tab, locate the `Tiered Storage` card and click
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: remove the rest

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 4]
```diff
@@ -38,9 +35,181 @@ console.
 
 </Procedure>
 
-After tiered storage is enabled you must either [manually tier data][manual-tier-chunk] or [setup a tiering policy][creating-data-tiering-policy] 
-to begin tiering data from your hypertables.
+## Automate tiering with policies
+
+A tiering policy automatically moves data to the object storage tier. Any chunks that only contain data
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: A tiering policy automatically moves any  data chunks that only contain data
older than the `move_after` threshold to the object storage tier.

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 5]
```diff
@@ -38,9 +35,181 @@ console.
...
+older than the `move_after` threshold are moved. This works similarly to a
+[data retention policy][data-retention], but chunks are moved rather than deleted. You can add tiering policies to hypertables, including continuous aggregates.
+
+A tiering policy schedules a job that runs periodically to migrate eligible chunks. The migration is asynchronous. The chunks are considered tiered once they appear in the `timescaledb_osm.tiered_chunks` view. Tiering does not influence your ability to query the chunks.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]:
```suggestion
A tiering policy schedules a job that runs periodically to asynchronously migrate eligible chunks to object storage. Chunks are considered tiered once they appear in the `timescaledb_osm.tiered_chunks` view. Tiering does not influence your ability to query the chunks.
```

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 6]
```diff
@@ -38,9 +35,181 @@ console.
...
+
+### Add a tiering policy
+
+To add a tiering policy, call `add_tiering_policy`:
+
+```sql
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md:59]: remove

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 7]
```diff
@@ -38,9 +35,181 @@ console.
...
+
+A tiering policy automatically moves data to the object storage tier. Any chunks that only contain data
+older than the `move_after` threshold are moved. This works similarly to a
+[data retention policy][data-retention], but chunks are moved rather than deleted. You can add tiering policies to hypertables, including continuous aggregates.
+
+A tiering policy schedules a job that runs periodically to migrate eligible chunks. The migration is asynchronous. The chunks are considered tiered once they appear in the `timescaledb_osm.tiered_chunks` view. Tiering does not influence your ability to query the chunks.
+
+### Add a tiering policy
+
+To add a tiering policy, call `add_tiering_policy`:
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md:57]:
```suggestion
To add a tiering policy, connect to your ${SERVICE_SHORT} and call `add_tiering_policy`:

 ![Add a tiering policy](https://assets.timescale.com/docs/images/console-add-tiering-policy.png)
```

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 8]
```diff
@@ -38,9 +35,181 @@ console.
...
+SELECT remove_tiering_policy(hypertable REGCLASS, if_exists BOOL = false);
+```
+
+For example, remove the tiering policy from the `example` hypertable in the following way:
+
+```sql
+SELECT remove_tiering_policy('example');
+```
+
+If you remove a tiering policy, new scheduled chunks will not be tiered. However, already tiered chunks won't be untiered. You can [untier chunks manually](#manually-tier-and-untier-chunks) to the local storage.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]:
```suggestion
If you remove a tiering policy, new scheduled chunks are not be tiered. However, chunks in tiered storage are not untiered. You [untier chunks manually](#manually-tier-and-untier-chunks) to local storage.
```

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 9]
```diff
@@ -38,9 +35,181 @@ console.
...
+
+## Manually tier and untier chunks
+
+If tiering policies do not meet your current needs, you can tier and untier chunks manually. 
+
+### Tier chunks
+
+Tiering a chunk is an asynchronous process that schedules the chunk to be tiered. In this example, you use a hypertable called `example` and tier chunks older than three days. You then proceed to list tiered chunks.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: in the following example, you tier chunks older than three days in the `example` [hypertable](link to hypertables). You then list the ...

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 10]
```diff
@@ -38,9 +35,181 @@ console.
...
+
+<Procedure>
+
+1. At the psql prompt, select all chunks in the table `example` that are older
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: Connect to your service using console or psql

or something along those lines. 

example hypertable

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 11]
```diff
@@ -38,9 +35,181 @@ console.
...
+
+1. To see which chunks are tiered into the object storage tier, use the `tiered_chunks` informational view:
+
+    ```sql
+    SELECT * FROM timescaledb_osm.tiered_chunks;
+    ```
+
+</Procedure>
+
+Tiering a chunk schedules it for migration to the object storage tier, but the migration won't happen immediately. Chunks are tiered one at a time in order to minimize database resource consumption. You can continue to query a chunk during migration.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: Can you add this to the procedure please.

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 12]
```diff
@@ -38,9 +35,181 @@ console.
...
 
+To see which chunks are scheduled for tiering either by policy or by a manual call, but have not yet been tiered, use this view:
+
+```sql
+SELECT * FROM timescaledb_osm.chunks_queued_for_tiering ;
+```
+
+### Untier chunks
+
+Tiered data is immutable. To update data in a tiered chunk, move it back to local storage, that is, Timescale's standard high-performance storage tier. You can do so by using the `untier_chunk` stored procedure.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: To update data in a tiered chunk, move it back to the standard high-performance storage tier in ${CLOUD_LONG}.

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 13]
```diff
@@ -38,9 +35,181 @@ console.
...
+
+Untiering chunks is a synchronous process. Chunks are renamed when the data is untiered.
+
+<Procedure>
+
+1.  At the `psql` prompt, check which chunks are currently tiered:
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: sql editor bla bla

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 14]
```diff
@@ -38,9 +35,181 @@ console.
...
+
+
+## Disable tiering 
+
+If you no longer want to use tiered storage for a particular hypertable, you
+can drop the associated metadata by calling the `disable_tiering` function.
+
+<Procedure>
+
+1. Call `remove_tiering_policy` and drop any tiering policy associated with this hypertable.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]:
```suggestion
SQL editor bla bla 

1. To drop all tiering policies associated with a table, call `remove_tiering_policy`.
```

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 15]
```diff
@@ -9,27 +9,24 @@ cloud_ui:
...
 
-# Tier data to the object storage tier
+# Manage tiering
 
-Enable tiered storage to begin migrating rarely used data from Timescale's standard high-performance storage tier
-to the object storage tier to save on storage costs. 
+You use tiered storage to save on storage costs. Specifically, you can migrate rarely used data from Timescale's standard high-performance storage to the object storage. With tiered storage enabled, you then either create automated tiering policies or manually tier and untier data.
 
-## Enabling the object storage tier
+Data on the object storage tier cannot be modified - so inserts, updates, and deletes will not work on tiered data. Make sure that you are not tiering data that is being <b>actively modified</b> to the object storage tier.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: Add a not saying this is for the Scale and Enterprise tiers only. Andddddd, please use the tiering widget in the page. Yaya

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Few comments, good stuff.

File: use-timescale/data-tiering/enabling-data-tiering.md (hunk 6 above)
Comment (atovpeko) [reply use-timescale/data-tiering/enabling-data-tiering.md:59]: remove what?

File: use-timescale/data-tiering/enabling-data-tiering.md (hunk 8 above)
Comment (atovpeko) [reply use-timescale/data-tiering/enabling-data-tiering.md]: This suggestion reverses the meaning

File: use-timescale/data-tiering/enabling-data-tiering.md (hunk 6 above)
Comment (billy-the-fish) [reply use-timescale/data-tiering/enabling-data-tiering.md:59]: The bits that are moved to the updates i suggested previously.

File: use-timescale/data-tiering/enabling-data-tiering.md (hunk 8 above)
Comment (billy-the-fish) [reply use-timescale/data-tiering/enabling-data-tiering.md]: You are right, updated.

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 16]
```diff
@@ -1,46 +1,222 @@
...
+
+Data on the object storage tier cannot be modified - so inserts, updates, and deletes will not work on tiered data. Make sure that you are not tiering data that is being <b>actively modified</b> to the object storage tier.
+
+## Enable tiered storage
+
+You enable tiered storage from the `Overview` tab in Console.
+
+<Procedure>
+
+1. In [Timescale Console][console], select the service to modify.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: Bold for the steps please.

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 17]
```diff
@@ -1,46 +1,222 @@
...
+   ```sql
+   select * from timescaledb_osm.tiered_hypertables;
+   ```
+
+</Procedure>
 
-[manual-tier-chunk]: /use-timescale/:currentVersion:/data-tiering/manual-tier-chunk/
-[creating-data-tiering-policy]: /use-timescale/:currentVersion:/data-tiering/creating-data-tiering-policy/
+[data-retention]: /use-timescale/:currentVersion:/data-retention/
+[console]: https://console.cloud.timescale.com/
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: Can you update to https://console.cloud.timescale.com/dashboard/services so it takes the user directly to the services page please.

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 18]
```diff
@@ -1,46 +1,222 @@
...
 
-# Tier data to the object storage tier
+# Manage tiering
 
-Enable tiered storage to begin migrating rarely used data from Timescale's standard high-performance storage tier
-to the object storage tier to save on storage costs. 
+You use tiered storage to save on storage costs. Specifically, you can migrate rarely used data from Timescale's standard high-performance storage to the object storage. After you enable tiered storage, you then either create automated tiering policies or manually tier and untier data.
 
-## Enabling the object storage tier
+<Highlight type="note">
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: Can you move this to under `When tiered storage is activated` please

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 19]
```diff
@@ -1,46 +1,222 @@
...
+
+   When tiered storage is activated, you see the amount of data in the tiered object storage.
+
+</Procedure>
+
+## Automate tiering with policies
+
+A tiering policy automatically moves any data chunks that only contain data
+older than the `move_after` threshold to the object storage tier. This works similarly to a
+[data retention policy][data-retention], but chunks are moved rather than deleted. You can add tiering policies to hypertables, including continuous aggregates.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: Can you add links to hypertables and continuous aggregates please.

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 20]
```diff
@@ -1,46 +1,222 @@
...
+
+### Add a tiering policy
+
+To add a tiering policy, connect to your $SERVICE_SHORT and call `add_tiering_policy`:
+
+```sql
+SELECT add_tiering_policy(hypertable REGCLASS, move_after INTERVAL, if_not_exists BOOL = false);
+```
+
+For example, tier chunks that are more than three days old in the `example` hypertable in the following way:
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]:
```suggestion
For example, to tier chunks that are more than three days old in the `example` hypertable: 
```

I know, it is not super google friendly

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 21]
```diff
@@ -1,46 +1,222 @@
...
+
+### Remove a tiering policy
+
+To remove an existing tiering policy, call `remove_tiering_policy`:
+
+```sql
+SELECT remove_tiering_policy(hypertable REGCLASS, if_exists BOOL = false);
+```
+
+For example, remove the tiering policy from the `example` hypertable in the following way:
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]:
```suggestion
For example, to remove the tiering policy from the `example` hypertable:
```

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 22]
```diff
@@ -1,46 +1,222 @@
...
+
+To see which chunks are scheduled for tiering either by policy or by a manual call, but have not yet been tiered, use this view:
+
+```sql
+SELECT * FROM timescaledb_osm.chunks_queued_for_tiering ;
+```
+
+### Untier chunks
+
+Tiered data is immutable. To update data in a tiered chunk, move it back to local storage, that is, Timescale's standard high-performance storage tier. You can do so by using the `untier_chunk` stored procedure.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: storage. That is, 

The storage tier is in Timescale Cloud, Timescale is a company. 

To untier a chunk, call the `untier_chunk` stored procedure.

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 23]
```diff
@@ -1,46 +1,222 @@
...
+
+Untiering chunks is a synchronous process. Chunks are renamed when the data is untiered.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: Don't see this should be on a newline. 

Can you add a lead in sentence before the procedure please.

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 24]
```diff
@@ -1,46 +1,222 @@
...
 
 </Procedure>
 
-After tiered storage is enabled you must either [manually tier data][manual-tier-chunk] or [setup a tiering policy][creating-data-tiering-policy] 
-to begin tiering data from your hypertables.
 
+## Disable tiering 
+
+If you no longer want to use tiered storage for a particular hypertable, you
+can drop the associated metadata by calling the `disable_tiering` function.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: hypertable, drop the ...

File: use-timescale/data-tiering/enabling-data-tiering.md (hunk 18 above)
Comment (atovpeko) [reply use-timescale/data-tiering/enabling-data-tiering.md]: moved. does not make sense to me to give the steps first and only explain why it does not work as an afterthought

File: use-timescale/data-tiering/enabling-data-tiering.md (hunk 20 above)
Comment (atovpeko) [reply use-timescale/data-tiering/enabling-data-tiering.md]: Well exactly

**Comment by billy-the-fish** [comment]
@gayyappan can you have a look at https://docs-dev.timescale.com/docs-3508-docs-rfc-reorg-tiering-policy-sections-into-manage-tiering/use-timescale/3508-docs-rfc-reorg-tiering-policy-sections-into-manage-tiering/data-tiering/enabling-data-tiering/ please. Any comments in this PR. 

@atovpeko: I have bad news for you. Now we have https://docs-dev.timescale.com/docs-3508-docs-rfc-reorg-tiering-policy-sections-into-manage-tiering/use-timescale/3508-docs-rfc-reorg-tiering-policy-sections-into-manage-tiering/data-tiering/enabling-data-tiering/ , I don't think we need https://docs-dev.timescale.com/docs-3508-docs-rfc-reorg-tiering-policy-sections-into-manage-tiering/use-timescale/3508-docs-rfc-reorg-tiering-policy-sections-into-manage-tiering/data-tiering/tour-data-tiering/ any more. @gayyappan, what do you think?

Do we have a corresponding redirects PR?

File: use-timescale/data-tiering/index.md [hunk 25]
```diff
@@ -42,30 +42,33 @@ solutions to offload data to secondary storage and fetch it back in when needed.
...
-* Take a [tour of tiered storage features][tour-data-tiering].
-* [Learn how to enable the object storage tier][enabling-data-tiering] on your service.
-* Manually [tier chunks][manual-tier-chunk] to schedule individual chunks to be tiered.
-* Create a [Tiering Policy][creating-data-tiering-policy] to automatically schedule chunks to be tiered.
-* [Learn how to query tiered data][querying-tiered-data].
-* Manually [untier chunks][untier-data] to move data back to the high-performance local storage tier.
-* [Disable tiering on a hypertable][disabling-data-tiering] on an individual table if you no longer want to associate it with tiered storage.
+This section explains the following:
+* [Learn about the object storage tier][about-data-tiering]: understand tiered storage before you 
+  [Manage tiering][enabling-data-tiering].
```
Comment (gayyappan) [inline use-timescale/data-tiering/index.md]: should remove this.

File: use-timescale/data-tiering/index.md [hunk 26]
```diff
@@ -42,30 +42,33 @@ solutions to offload data to secondary storage and fetch it back in when needed.
...
+* [Tour tiered storage][tour-data-tiering]: see the different features in tiered storage. 
+* [Manage tiering][enabling-data-tiering]: enable and disable data tiering, automate tiering with 
+   policies or tier and untier manually.
+* [Query tiered data][querying-tiered-data]: query and performance for tiered data.
+* [Replicas and forks with tiered data][replicas-and-forks]: billing and tiered storage.
```
Comment (gayyappan) [inline use-timescale/data-tiering/index.md]:
```suggestion
* [Replicas and forks with tiered data][replicas-and-forks]: How does tiered storage work with forks and replicas. 
```

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 27]
```diff
@@ -1,46 +1,221 @@
...
 
-After tiered storage is enabled you must either [manually tier data][manual-tier-chunk] or [setup a tiering policy][creating-data-tiering-policy] 
-to begin tiering data from your hypertables.
+## Automate tiering with policies
+
+A tiering policy automatically moves any chunks that only contain data
+older than the `move_after` threshold to the object storage tier. This works similarly to a
+[data retention policy][data-retention], but chunks are moved rather than deleted. 
+
+A tiering policy schedules a job that runs periodically to asynchronously migrate eligible chunks to object storage. Chunks are considered tiered once they appear in the `timescaledb_osm.tiered_chunks` view.
```
Comment (gayyappan) [inline use-timescale/data-tiering/enabling-data-tiering.md:51]:
```suggestion
A tiering policy schedules a job that runs periodically to asynchronously migrate eligible chunks to object storage.  After chunks are tiered, they appear in the `timescaledb_osm.tiered_chunks` view. 
```

**Review by gayyappan — COMMENTED** [review]
minor edits suggested.
The overall "Manage tiering" section looks good!

File: use-timescale/data-tiering/index.md [hunk 28]
```diff
@@ -42,30 +42,33 @@ solutions to offload data to secondary storage and fetch it back in when needed.
...
-* Take a [tour of tiered storage features][tour-data-tiering].
-* [Learn how to enable the object storage tier][enabling-data-tiering] on your service.
-* Manually [tier chunks][manual-tier-chunk] to schedule individual chunks to be tiered.
-* Create a [Tiering Policy][creating-data-tiering-policy] to automatically schedule chunks to be tiered.
-* [Learn how to query tiered data][querying-tiered-data].
-* Manually [untier chunks][untier-data] to move data back to the high-performance local storage tier.
-* [Disable tiering on a hypertable][disabling-data-tiering] on an individual table if you no longer want to associate it with tiered storage.
+This section explains the following:
+* [Learn about the object storage tier][about-data-tiering]: understand tiered storage.
+* [Tour tiered storage][tour-data-tiering]: see the different features in tiered storage.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/index.md]: Can you remove this link please.

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 29]
```diff
@@ -1,46 +1,222 @@
...
 keywords: [tiered storage]
 tags: [storage, data management]
 cloud_ui:
     path:
         - [services, :serviceId, overview]
+plans: [scale, enterprise]  
 ---
 
-# Tier data to the object storage tier
+# Manage tiering
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]: Maybe the title should explain more clearly what we explain. Manage automatic and manual tiering?

File: use-timescale/data-tiering/about-data-tiering.md [hunk 30]
```diff
@@ -7,55 +7,82 @@ tags: [storage, data management]
 cloud_ui:
     path:
         - [services, :serviceId, overview]
+plans: [scale, enterprise]  
 ---
 
 # About the object storage tier
 
-The tiered storage architecture complements Timescale's standard high-performance storage tier with a low-cost object storage tier.
+The Timescale's tiered storage architecture includes a standard high-performance storage tier and a low-cost object storage tier built on Amazon S3. You can use the standard tier for data that requires quick access, and the object tier for rarely used historical data. Chunks from a single hypertable, including compressed chunks, can stretch across these two storage tiers. A compressed chunk uses a different storage representation after tiering.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/about-data-tiering.md]:
```suggestion
Timescale's tiered storage architecture includes a standard high-performance storage tier, and a low-cost object storage tier built on Amazon S3. You use the standard tier for data that requires quick access, and the object tier for rarely used historical data. Chunks from a single hypertable, including compressed chunks, can stretch across these two storage tiers. A compressed chunk uses a different storage representation after tiering.
```

File: use-timescale/data-tiering/about-data-tiering.md [hunk 31]
```diff
@@ -7,55 +7,82 @@ tags: [storage, data management]
...
-You can move your hypertable data across the different storage tiers to get the best price performance.
-You can use the standard high-performance storage tier for data that requires quick access,
-and the low-cost object storage tier for rarely used historical data. 
-Regardless of where your data is stored, you can still query it with
-[standard SQL][querying-tiered-data].  
-Because it's queried normally with SQL, you can still JOIN against tiered data, 
-build views on tiered data, and even define continuous aggregates on tiered data.
-In fact, because the implementation of continuous aggregates also use hypertables, 
-they can be tiered to low-cost storage as well.
+In the standard storage, chunks are stored in the block format. In the object storage, they are stored in a compressed, columnar format. This format is different from that of the internals of the database, for better interoperability across various platforms. It allows for more efficient columnar scans across longer time periods, and Timescale uses other metadata and query optimizations to reduce the amount of data that needs to be fetched from the object storage tier to satisfy a query.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/about-data-tiering.md]:
```suggestion
In high-performance storage, chunks are stored in the block format. In the object storage, they are stored in a compressed, columnar format. For better interoperability across various platforms, this format is different from that of the internals of the database. It allows for more efficient columnar scans across longer time periods, and Timescale Cloud uses other metadata and query optimizations to reduce the amount of data that needs to be fetched from the object storage tier to satisfy a query. 
```

File: use-timescale/data-tiering/about-data-tiering.md [hunk 32]
```diff
@@ -7,55 +7,82 @@ tags: [storage, data management]
...
-*   **Online.**  Your data is always there and can be [queried when needed][querying-tiered-data]. 
+*   **Online:** your data is always there and can be [queried when needed][querying-tiered-data]. 
 
 ## Architecture
 
-The tiered storage backend works by periodically and asynchronously moving older chunks to the object storage tier;
-an object store built on Amazon S3.
-There, it's stored in the Apache Parquet format, which is a compressed
-columnar format well-suited for S3. Data remains accessible both during and after the migration.
+The tiered storage backend works by periodically and asynchronously moving older chunks to the object storage tier.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/about-data-tiering.md]:
```suggestion
The tiered storage backend works by periodically and asynchronously moving older chunks from high-performance storage to the object storage tier. 
```

File: use-timescale/data-tiering/about-data-tiering.md [hunk 33]
```diff
@@ -7,55 +7,82 @@ tags: [storage, data management]
...
-pulls data from wherever it's located: the standard high-performance storage tier, the object storage tier, or both.
 Various SQL optimizations limit what needs to be read from S3:
 
-*   Chunk exclusion avoids processing chunks that fall outside the query's time window
-*   The database uses metadata about row groups and columnar offsets, so only
-    part of an object needs to be read from S3
-
-The result is transparent queries across standard PostgreSQL storage and S3
-storage, so your queries fetch the same data as before.
+* Chunk pruning - exclude the chunks that fall outside the query time window.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/about-data-tiering.md]: Can you put **Chunk pruning**: etc in bold to match the other lists in the page please.

File: use-timescale/data-tiering/about-data-tiering.md [hunk 34]
```diff
@@ -7,55 +7,82 @@ tags: [storage, data management]
...
+* Row group pruning - identify the row groups within the Parquet object that satisfy the query.
+* Column pruning - fetch only columns that are requested by the query.
+
+The result is transparent queries across standard PostgreSQL storage and S3 storage, so your queries fetch the same data as before.
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/about-data-tiering.md]:
```suggestion
The result is transparent queries across high-performance storage and S3  object storage , so your queries fetch the same data as before.
```

File: use-timescale/data-tiering/enabling-data-tiering.md [hunk 35]
```diff
@@ -1,46 +1,222 @@
...
 
-Enable tiered storage to begin migrating rarely used data from Timescale's standard high-performance storage tier
-to the object storage tier to save on storage costs. 
+You use tiered storage to save on storage costs. Specifically, you can migrate rarely used data from Timescale's standard high-performance storage to the object storage. After you [enable tiered storage](#enable-tiered-storage), you then either [create automated tiering policies](#automate-tiering-with-policies) or [manually tier and untier data](#manually-tier-and-untier-chunks).
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/enabling-data-tiering.md]:
```suggestion
You use tiered storage to save on storage costs. Specifically, you can migrate rarely used data from Timescale's standard high-performance storage to object storage. After you [enable tiered storage](#enable-tiered-storage), you then either [create automated tiering policies](#automate-tiering-with-policies) or [manually tier and untier data](#manually-tier-and-untier-chunks).
```

File: use-timescale/data-tiering/querying-tiered-data.md [hunk 36]
```diff
@@ -23,95 +21,170 @@ sessions.
 With tiered reads enabled, you can query your data normally even when it's distributed across different storage tiers.
 Your hypertable is spread across the tiers, so queries and `JOIN`s work and fetch the same data as usual.
 
-<!-- vale Google.Acronyms = YES -->
-
 <Highlight type="warning">
```
Comment (billy-the-fish) [inline use-timescale/data-tiering/querying-tiered-data.md]: I'd make this into a sentence without the warning and link to the performance section.  if you must, make it an info admomition.

**Review by billy-the-fish — COMMENTED** [review]
These pages are really coming together.

**Comment by atovpeko** [comment]
@iroussos could you please take a look at all those pages in Manage tiering, rather than just the diff, to get a bigger picture?

**Comment by iroussos** [comment]
> @iroussos could you please take a look at all those pages in Manage tiering, rather than just the diff, to get a bigger picture?

Makes sense, thank you @atovpeko!

**Review by iroussos — APPROVED** [review]
Thank you @atovpeko, looks great to me!

We should publish as is in my opinion - today. And we can come back to refine if needed (not that I can find anything to add on my side)

**Review by billy-the-fish — APPROVED** [review]

**Pull request merged**